system/Router/RouteCollection.php

```diff
@@ -1393,6 +1393,7 @@ public function resetRoutes()
         }
 
         $this->prioritizeDetected = false;
+        $this->didDiscover        = false;
```

Comment:
This seems like it could be a breaking change, though I'm not sure why this wasn't the case before.